new-frontiers-14/frontier-station-14
# Apothecary Medical Assembler Update

## About the PR



Added:
- new medical assembler.
- basic materials wall locker in the cargo room for the medical techfab.
- locked buttons in cockpit and medical ward. Cockpit for thruster power control, ward for door bolts.

Tweaked:
- Updated medical signs with EMS signs. 
- Moved charger to L reception table to make room for medical assembler.
- Upped power setting on generator to 17kw to accommodate the power draw from the medical assembler.
- Replaced upstream medical and chemistry wall lockers with new Erhard visual filled versions.

Removed:
- One EVA suit locker from airlock.
- Chem lab coat.

## Why / Balance


Medical assembler good.
Taking advantage of new visual lockers and gameplay mechanics to update and maintain my ship. As one does.

## How to test


Check that all lockers can be accessed.
Check functionality of all new buttons.
Ensure medical assembler is powered.

## Media


![ss+(2024-10-14+at+08 00 26)](https://github.com/user-attachments/assets/11eef252-ea9e-4d1f-9f9e-0ac4a8d29bb6)



## Requirements



## Breaking changes


**Changelog**


🆑 
- add: Apothecary now has the Medical Assember.
